src/env.h

```diff
@@ -329,32 +330,49 @@ constexpr size_t kFsStatsBufferLength = kFsStatsFieldsNumber * 2;
   V(x_forwarded_string, "x-forwarded-for")                                     \
   V(zero_return_string, "ZERO_RETURN")
 
-#define ENVIRONMENT_STRONG_PERSISTENT_PROPERTIES(V)                            \
-  V(as_callback_data, v8::Object)                                              \
+#define ENVIRONMENT_STRONG_PERSISTENT_DATA(V)                                  \
   V(as_callback_data_template, v8::FunctionTemplate)                           \
+  V(async_wrap_ctor_template, v8::FunctionTemplate)                            \
+  V(async_wrap_object_ctor_template, v8::FunctionTemplate)                     \
+  V(context, v8::Context)                                                      \
```

Comment:
Does `ENVIRONMENT_STRONG_PERSISTENT_DATA` imply they inherit from v8::Data? Because v8::Context doesn't.

Reply:
hmm, right, I guess I should just rename them to `PROPERTIES` (can't really think of a better name)